cpp/include/cudf/types.hpp

```diff
@@ -292,5 +292,7 @@ enum class hash_id {
   HASH_MD5            ///< MD5 hash function
 };
 
+enum class flip_endianness : bool { NO, YES };
```

Comment:
This seems pretty specific to this API and should probably just be in the same header.